### DISCUSSION TITLE
How to run the cache web api

### DISCUSSION THREAD

**Discussion by swfz** [denoland/deno#19873] [Q&A] [answered]
https://deno.com/blog/v1.26#cache-web-api

I saw this release and tried to deploy to deno to actually try the Cache Web API, but it doesn't seem to work with an error

```
ReferenceError: caches is not defined
```

Is there anything wrong with this condition?

**Comment by boywithkeyboard** [comment, marked as answer]
Hey @swfz!

So you're trying to use the Web Cache API on Deno Deploy. I asked Ryan the same question a while ago and he told me that they plan to eventually release it on Deno Deploy, but other features would have a higher priority at the moment.

The linked changelog barely states that the Cache API is included in the Deno runtime starting with v1.26 & bound to the local file-system.

Sorry, but you're out of luck here!

**Reply by swfz** [reply]
Thanks for the reply.
I see you were planning to do this in the future, I'll be waiting in anticipation.

**Comment by markthree** [comment]
> https://deno.com/blog/v1.26#cache-web-api
> 
> I saw this release and tried to deploy to deno to actually try the Cache Web API, but it doesn't seem to work with an error我看到了这个版本，并试图部署到deno实际尝试该高速缓存Web API，但它似乎没有工作与错误
> 
> ```
> ReferenceError: caches is not defined
> ```
> 
> Is there anything wrong with this condition?

This is also a problem encountered by Fresh, a disguised approach is to use deno kv，Relevant issues visible 👉 [fresh/issues/1062](https://github.com/denoland/fresh/issues/1062)，I hope it's helpful to you

**Comment by brc-dd** [comment]
This is now supported on deno deploy - https://deno.com/blog/deploy-cache-api

The docs mention it's "closed beta" though. So, it might take bit more time to be generally available - https://docs.deno.com/deploy/manual/edge-cache/
